### ISSUE TITLE
Fix Tensorflow import errors

### ISSUE THREAD

**Issue by svenvanderburg** [matchms/ms2deepscore#99]
See [these linter results](https://github.com/matchms/ms2deepscore/runs/5476610429?check_suite_focus=true)

It's probably related to a new version of Tensorflow (2.8.0).

```
ms2deepscore/data_generators.py
  Line: 8
    pylint: import-error / Unable to import 'tensorflow.keras.utils'
  Line: 96
    pylint: consider-using-f-string / Formatting a regular string which could be a f-string (col 29)

ms2deepscore/models/SiameseModel.py
  Line: 5
    pylint: import-error / Unable to import 'tensorflow.keras.layers'
```

**Comment by florian-huber** [comment]
The keras import issue also happend with some past versions of tensorflow, so it could again be the same thing (see https://github.com/tensorflow/tensorflow/issues/26502). I will work fine at runtime but the linter oder editors will complain because keras is not in the tensorflow __init__.

We should probably switch to `from tensorflow.python import keras` or similar.

**Comment by florian-huber** [comment]
I was wrong. Switching to tensorflow.python is not the way to go. I will try to simply silence the pylint import warnings since it seems to be tensorflow internal issue.

**Comment by florian-huber** [comment]
Should be fixed with #101
